ROADMAP.md

```diff
@@ -0,0 +1,123 @@
+xk6-browser roadmap
+===================
+
+xk6-browser is a young project that is undergoing substantial changes in order to make it more useful to all users—whether they're coming from API/performance testing with standard [k6](https://k6.io/), or are new to the k6 ecosystem and browser testing in general. In addition to being a useful tool, users would want to know what the team's, k6, and Grafana's goals are for the project, so that they can plan ahead and trust our commitment to its success.
+
+With that in mind, below we detail some of our short, mid and long-term goals. Most of these will be worked on concurrently, and reaching them will be a gradual process. The timeframes are also not set in stone, but serve as a tentative target the team is aiming for.
+
+
+Short-term goals
+----------------
+
+These are goals achievable within 3-6 months, tentatively done by Q3 2022.
+
+- **Stabilize the existing functionality and fix any major, high-impact and blocking bugs.**<br>
+  Tool stability is something we take very seriously and users shouldn't run into any major showstopping issues.
+
+  *How will we achieve this?*<br>
+  By manually testing the application in different scenarios, adding more automated tests, and addressing reports from internal and external users.
+  Any issues found that have a considerable user impact will be given highest priority and fixed ASAP.
+
+  *Definition of Done*<br>
+  This is difficult to gauge, but at some point we should stop running into any major issues on a frequent basis and should consider this goal as reached.
+
+
+- **Limited Alpha deployment and testing of the extension in k6 Cloud.**<br>
+
+  *How will we achieve this?*<br>
+  Soon we will create a small-scale deployment for internal use and specific customers, meant to test the basic integration of the extension in k6 Cloud.
+
+  *Definition of Done*<br>
+  When the extension is usable in k6 Cloud; i.e. scripts can run and test results are shown. Frontend changes are not required at this stage.
+
+
+- **Update [API documentation](https://k6.io/docs/javascript-api/xk6-browser/).**<br>
+  The current documentation is not up-to-date which makes using xk6-browser outside of the examples quite difficult.
+
+  *How will we achieve this?*<br>
+  We are prioritizing working on documentation in all upcoming cycles.
```

Comment:
> in all upcoming **development** cycles. 

I don't think our users know what our cycles are :-)

Reply:
Sure :smile: Changed in d17ee93.